tasks/src/test/java/webdriver/google_cloud/browser/Browser.java

```diff
@@ -0,0 +1,21 @@
+package webdriver.google_cloud.browser;
+
+import static webdriver.google_cloud.constants.Constants.*;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class Browser {
+
+  public static void waitForVisibleElement(WebDriver driver, WebElement element) {
+    new WebDriverWait(driver, TIME_OUT_IN_SECONDS)
+        .until(ExpectedConditions.visibilityOf(element));
+  }
+
+  public static void waitForVisibleFrame(WebDriver driver) {
+    new WebDriverWait(driver, TIME_OUT_IN_SECONDS)
+        .until(ExpectedConditions.frameToBeAvailableAndSwitchToIt(FRAME_LOCATOR));
```

Comment:
I'd not hardcode frame locator and make the method reusable for other frames extracting a parameter